wiki/docs/modules/request manager.md

```diff
@@ -0,0 +1,76 @@
+# Request Manager
+
+Request manager is used to process any request from viewmodel to respective repositories.
+
+
+
+## Usability
+
+1. Wrap the request inside the RequestManager object like below,
+
+   ```
+   object : RequestManager<User>(preCheck = {
```

Comment:
RequestManager<T> 
& we can mention here T can be class of interest for example "User" for login usecase